File: leet-code-solutions/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        seeker=0
        holder=0

        while seeker < len(nums):
            if nums[holder] == nums[seeker]:
                seeker+=1
            else:
                nums[holder + 1] = nums[seeker]
                holder += 1


        return holder + 1

File: Div3-Group1/RemoveDuplicatesfromSortedArray.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        for (int i = nums.size()-1; i >0; i--)
        {
            if (nums[i]==nums[i-1])
            {
                nums.erase(nums.begin() + i);
            }
            
        }
        return nums.size();
    }
};
